client/profile-wizard/steps/business-details/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { Spinner } from '@woocommerce/components';
+import { ONBOARDING_STORE_NAME, SETTINGS_STORE_NAME } from '@woocommerce/data';
+
+/**
+ * Internal dependencies
+ */
+import { BundleBusinessDetailsStep } from './flows/bundle';
+import { SelectiveFeaturesBusinessStep } from './flows/selective-bundle';
+import { getCountryCode } from '../../../dashboard/utils';
+import './style.scss';
+
+export const BusinessDetailsStep = ( props ) => {
+	const { profileItems, settings, isLoading } = useSelect( ( select ) => {
+		return {
+			isLoading:
+				! select( ONBOARDING_STORE_NAME ).hasFinishedResolution(
+					'getProfileItems'
+				) ||
+				! select(
+					SETTINGS_STORE_NAME
+				).hasFinishedResolution( 'getSettings', [ 'general' ] ),
+			profileItems: select( ONBOARDING_STORE_NAME ).getProfileItems(),
+			settings:
+				select( SETTINGS_STORE_NAME ).getSettings( 'general' ) || {},
+		};
+	} );
+
+	const country = settings.general
+		? settings.general.woocommerce_default_country
+		: null;
+
+	const industrySlugs = ( profileItems.industry || [] ).map(
+		( industry ) => industry.slug
+	);
+
+	const selectiveBundleInstallSegmentation =
+		getCountryCode( country ) === 'US' &&
+		( industrySlugs.includes( 'food-drink' ) ||
+			industrySlugs.includes( 'other' ) );
+
+	if ( isLoading ) {
+		return (
+			<div className="woocommerce-admin__business-details__spinner">
+				<Spinner />
+			</div>
+		);
+	}
+
+	if ( selectiveBundleInstallSegmentation ) {
```

Comment:
segmentation determines if we render the new step I've introduced. But there is existing segmentation for another experiment inside `< BundleBusinessDetailsStep>` that will determine what is rendered there. It was far simpler to leave that segmentation in place than try to bring it here. I hope in future this will all get consolidated to a single step or we will have a proper A/B test framework in place.